platform/ios/app/MBXViewController.m

```diff
@@ -2030,6 +2029,21 @@ - (void)setStyles
     [self.styleNames addObject:@"MapLibre Basic"];
     [self.styleURLs addObject:[NSURL URLWithString:@"https://demotiles.maplibre.org/style.json"]];
 
+    [self.styleNames addObject:@"US"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"http://data.local/us-map-style.json"]];
+
+    [self.styleNames addObject:@"Americana"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"https://zelonewolf.github.io/openstreetmap-americana/style.json"]];
+
+    [self.styleNames addObject:@"Facebook Light"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"https://external.xx.fbcdn.net/maps/vt/style/canterbury_1_0/?locale=en_US"]];
+
+    [self.styleNames addObject:@"Facebook Dark"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"https://external.xx.fbcdn.net/maps/vt/style/dark/?locale=en_US"]];
+
+    [self.styleNames addObject:@"Facebook 3D"];
+    [self.styleURLs addObject:[NSURL URLWithString:@"https://external.xx.fbcdn.net/maps/vt/style/default_3d/?locale=en_US"]];
```

Comment:
We shouldn't be adding these endpoints here (licensing complexities) - we can probably provide an offline package for the MapLibre organization to self-host however.